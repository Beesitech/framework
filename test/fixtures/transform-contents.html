<div foo>foo</div>
<div bar>bar</div>
