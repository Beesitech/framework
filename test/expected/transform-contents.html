<div>Foo is the way</div>
<div>Bar all day</div>
